lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -37,4 +35,15 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
   @openUploadLogoForm = ->
     ModalService.open LogoPhotoForm, group: => @group
 
+  @handleSubscriptionSuccess = ->
```

Comment:
This is great for now, but this 'do stuff when query parameters are there' feels like a separate concern to me, and I like the idea of separating it out. Haven't got a great idea for how yet tho.